sdk/network/Azure.ResourceManager.Network/src/Generated/BastionHostResource.cs

```diff
@@ -221,32 +222,32 @@ public virtual ArmOperation Delete(WaitUntil waitUntil, CancellationToken cancel
         }
 
         /// <summary>
-        /// Creates or updates the specified Bastion Host.
+        /// Updates Tags for BastionHost resource
         /// <list type="bullet">
         /// <item>
         /// <term>Request Path</term>
         /// <description>/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/bastionHosts/{bastionHostName}</description>
         /// </item>
         /// <item>
         /// <term>Operation Id</term>
-        /// <description>BastionHosts_CreateOrUpdate</description>
+        /// <description>BastionHosts_UpdateTags</description>
         /// </item>
         /// </list>
         /// </summary>
         /// <param name="waitUntil"> <see cref="WaitUntil.Completed"/> if the method should wait to return until the long-running operation has completed on the service; <see cref="WaitUntil.Started"/> if it should return after starting the operation. For more information on long-running operations, please see <see href="https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/samples/LongRunningOperations.md"> Azure.Core Long-Running Operation samples</see>. </param>
-        /// <param name="data"> Parameters supplied to the create or update Bastion Host operation. </param>
+        /// <param name="networkTagsObject"> Parameters supplied to update BastionHost tags. </param>
         /// <param name="cancellationToken"> The cancellation token to use. </param>
-        /// <exception cref="ArgumentNullException"> <paramref name="data"/> is null. </exception>
-        public virtual async Task<ArmOperation<BastionHostResource>> UpdateAsync(WaitUntil waitUntil, BastionHostData data, CancellationToken cancellationToken = default)
+        /// <exception cref="ArgumentNullException"> <paramref name="networkTagsObject"/> is null. </exception>
+        public virtual async Task<ArmOperation<BastionHostResource>> UpdateAsync(WaitUntil waitUntil, NetworkTagsObject networkTagsObject, CancellationToken cancellationToken = default)
```

Comment:
There are breaking changes in the interfaces.
- Add manual codes to provide backward compatibility, see compute for reference: https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/compute/Microsoft.Azure.Management.Compute/src/Customizations
- By the way, there is already `SetTags` for this class: https://github.com/Azure/azure-sdk-for-net/blob/a9a133a89fdd2d3f4b40642776059b88a06c8345/sdk/network/Azure.ResourceManager.Network/src/Generated/BastionHostResource.cs#L413